### ISSUE TITLE
Magic Shortcuts for Markdown

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/web-components#321]
# Feature Request 🛍️

I like not having to move my cursor when I write text and one way to do that is to use markdown things in your text and have the editor automatically convert them as you write. Am I the only one who likes that kind of editing?

Demoed here on the editor from Slab: https://slab.com
![EditingShortcuts](https://user-images.githubusercontent.com/670099/112549683-8f131200-8d94-11eb-8d6d-4f0292223270.gif)

## Use Case
As a user who understands markdown, I like to be able to type markdown in the editor and have it converted for me to WYSIWYG.

## Possible Solution
Look at sequences of characters during editing and detect markdown things in 'em.

## Notes

The example above was written without moving the cursor to go to the toolbar, or having to select parts of the text, but only by typing.

**Comment by Cronus1007** [comment]
@jeromesimeon I would like to implement this one.

**Comment by d-e-v-esh** [comment]
This seems to be lots of small features. Should this be all in one go or should we go one by one?

**Comment by irmerk** [comment]
@d-e-v-esh I agree, this will be incremental. I think we should go one by one and use this Issue to track the changes.

**Comment by d-e-v-esh** [comment]
@jeromesimeon @irmerk I created a PR #288 a while ago that does exactly the same thing. Maybe we can build on top of that to implement these features after that is merged.

**Comment by irmerk** [comment]
To reiterate from my comment in #288, that's a good start to build on for this Issue.
